Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -179,6 +186,14 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
         name: .ksr_projectBacked,
         object: nil
       )
+
+    self.sessionStartedObserver = NotificationCenter.default
```

Comment:
Just curious why use an observer stored locally that you need to de-init? You probably don't need it (as done in `ProjectPageNavigationBarView`)

Reply:
Updated.